CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.15.0
+### Added
+- Support for project type `pimcore` in project type resolvers.
+- Separate configuration file for project type `magento2`.
```

Comment:
This reads like it qualifies as  "CHANGED" see https://keepachangelog.com/en/1.0.0/

```
Types of changes
Added: for new features.
Changed: for changes in existing functionality.
Deprecated: for soon-to-be removed features.
Removed: for now removed features.
Fixed: for any bug fixes.
Security: in case of vulnerabilities.
```

It could be a feature though, but then it's not clear how to use the feature (this could maybe be added to the README.md then)

Reply:
Cleaned up, thanks for the input :+1: